ui/lib/core/addon/components/navigate-input.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="navigate-filter">
+  <div class="field" data-test-nav-input>
+    <p class="control has-icons-left">
+      {{! template-lint-disable no-down-event-binding }}
+      <input
+        class="filter input"
+        value={{@filter}}
+        placeholder={{or @placeholder "Filter keys"}}
+        type="text"
+        data-test-comoponent="navigate-input"
+        {{on "input" this.handleInput}}
+        {{on "keyup" this.handleKeyUp}}
+        {{on "keydown" this.handleKeyPress}}
```

Comment:
we use the keydown for TAB. While this means I need the template-lint-disable above I don't want to remove this as it would impact functionality.

Reply:
I'm not sure on that answer. I think overall this component could use a refactor. When I noticed the age and round about pattern of the code, I focused on glimmerizing and avoided changing the flow (for fear I would break something). However, I'm open to suggestions if folks have any.

Reply:
ember [concurrency](https://embermap.com/topics/ember-concurrency/our-first-task-an-autocomplete-searchbox) has an example, but it's old and not really our pattern because we change the URL with params instead of making a network request (because we don't have flags for filtering ....)

![image](https://user-images.githubusercontent.com/6618863/224865476-eafdc574-abc6-423d-9649-a9523554d39d.png)

Reply:
Yeah - that's a good point. I think filtering via params here makes sense. Good find, though! I miss ember map and wish they'd do some newer stuff. 

I think that it probably makes sense to just start with glimmerization and do a bigger refactor another time, if that makes sense down the line.

Reply:
I think when it comes to glimmerizing some of these older components it can be complex enough just to update them, let alone audit how they function and refactor as well. I like keeping the glimmerization work separate from improving the functionality.